README.md

```diff
@@ -23,7 +23,19 @@ In your terminal, go to the project directory
 - Install Bundler if this gem is not installed yet :
 
 ```
-gem install Bundler
+gem install bundler
+```
+
+- Install cocoapods if this gem is not installed yet :
+
+```
+gem install cocoapods
+```
+
+- Make sure all cocoapods are up 2 date :
+
+```
+pod repo update
```

Comment:
the pod command needs to be executed by bundler to execute the local cocoapods installation
bundle exec pod repo update